alembic/testing/util.py

```diff
@@ -78,3 +80,11 @@ def run_ddl(self):
         return fixture_functions.fixture(scope=ddl)(run_ddl)
 
     return decorate
+
+
+def path(linuxPath):
+    """Takes a linux path with / and transforms it to the correct os path.
+    This is mainly to support windows \\"""
+    if win:
+        return join(*linuxPath.split("/"))
```

Comment:
doesnt.....os.path.normpath do this?

Reply:
Indeed, I remembered there was something that did this, but I did not remember what it was.